src/polyfills.js

```diff
@@ -4,6 +4,5 @@
  * @desc Add support for older browsers.
  */
 import '@babel/polyfill';
-import 'react-app-polyfill/ie11';
```

Comment:
was this an automatic change or manual? is there a directive regarding which browsers we need to support?

Reply:
this was a manual change, it was causing an error during deploy time because of a dependencies issue. I chatted with @thathert about it and we decided not to support IE since most people are using something else these days (chrome/firefox) and IE11 is reaching EoS from microsoft as they move towards the chromium based version of Edge